pkg/controller/backup_finalizer_controller.go

```diff
@@ -121,6 +121,7 @@ func (r *backupFinalizerReconciler) Reconcile(ctx context.Context, req ctrl.Requ
 			velerov1api.BackupPhaseFailed,
 			velerov1api.BackupPhaseFailedValidation:
 			r.backupTracker.Delete(backup.Namespace, backup.Name)
+			r.metrics.DecrementActiveBackupTotal()
```

Comment:
Wouldn't we be decrementing twice? If not, please add a comment which decrement is the common happy path case and which is fallback/failure etc.

Reply:
Yes, I think I was trying to match every decrement with every Delete, but I guess this one only runs if the backup didn't get deleted for some reason? That would cause extra decrements, so I will take this one out.